proto/event.proto

```diff
@@ -29,9 +29,15 @@ message Value {
 message Metric {
   oneof metric {
     Counter counter = 1;
-    Histogram histogram = 2;
-    Gauge gauge = 3;
+    Gauge gauge = 2;
+    Histogram histogram = 3;
```

Comment:
Changing the tag numbers here (`2` and `3` are swapped) makes this a breaking change when it doesn't really need to be.